API.md

```diff
@@ -1,26 +1,25 @@
 # API Reference
-Include and instantiate the TheThingsNetwork class. The constructor sets the value of the spreading factor, the front-side Bus and the frequency plan. It also initialize the library with the Streams it should communicate with.
+Include and instantiate the TheThingsNetwork class. The constructor initialize the library with the Streams it should communicate with. It also sets the value of the spreading factor, the front-side Bus and the frequency plan.
 
 ```c
 #include <TheThingsNetwork.h>
-TheThingsNetwork ttn(Stream& modemStream, Stream& debugStream, fp_ttn_t fp, int sf = 7, int fsb = 2);
+TheThingsNetwork ttn(Stream& modemStream, Stream& debugStream, fp_ttn_t fp = TTN_FP_EU868, int sf = 7, int fsb = 2);
```

Comment:
Don't initialize EU868 by default. The user has to specify it explicitly. We do not favor a region